task2/task2.php

```diff
@@ -0,0 +1,7 @@
+<?php
+function multipleAddition(int $x): int
+{
+    $number = array_sum(array_map('intval', str_split($x)));
+    return count($number) !== 1 ? multipleAddition($number) : $number;
```

Comment:
Why do you do reverse comparision in if-else block, if direct comparison is faster?